home/views.py

```diff
@@ -7,3 +8,30 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
```

Comment:
Not seeing any tests for this view

Reply:
I don't understand what you mean you don't see tests for this view.
Every test exists because I need it for the view:
* The answer feed function which queries the answers to display,
* The thumb up / down functions
* The question title I use in the view
* The HTTP status response
All of them don't have a purpose without the view display question I made.
What am I missing?

Reply:
I think the method you have above that checks the view through the client wasn't there the last time I looked